samples/graph-bulk-meetings/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-Meeting-Event-csharp",
+    "source": "officeDev",
+    "title": "This is a sample application that shows the usage of create training event meeting",
+    "shortDescription": "This is a sample application which demonstrates how to create multiple meeting training meeting event at one click (using excel sheet upload file).",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/v-keshsharma/eventcreation/samples/graph-meetings-bulks/csharp",
+    "longDescription": [
+      "This is a sample application which demonstrates how to create multiple meeting training meeting event at one click (using excel sheet upload file)."
```

Comment:
Change the desctiotion to
This is an sample application which demonstrates how to create multiple Teams meetings at one click (using excel sheet upload file).

Reply:
Resolved